docs/index.md

```diff
@@ -1,4 +1,4 @@
-[![](https://dietpi.com/images/Slider01.png){: style="height:180px"}](https://dietpi.com)
+[![](https://dietpi.com/images/logo_transparent_360x360.png){: style="height:180px"}](https://dietpi.com)
```

Comment:
Ah yes sorry for that, I wanted to give this a more reasonable name and cropped + resized it (from 920x720). If fixed 180px are needed, there are as well smaller versions, but I'll sort them a bid and adjust image sources then.